client/blocks/reader-full-post/_content.scss

```diff
@@ -331,6 +331,22 @@
 	}
 }
 
+// Longreads-Specific Full Post View Styles
+.blog-70135762 .reader-full-post__story-content {
+	
+	.publisher-intro {
+		img {
```

Comment:
Sure, I can do that. I'll also need to un-nest the other change on L342 then: `.publisher-intro p:first-child`.